nexus/auth/src/authz/context.rs

```diff
@@ -51,7 +52,6 @@ impl Authz {
         self.oso.is_allowed(actor.clone(), action, resource.clone())
     }
 
-    #[cfg(test)]
```

Comment:
The `policy_test` (look for the files changed in this PR) was previously a "unit test" in `db-queries` that used auth and datastore methods extensively.

It has stayed in `db-queries` to access the database methods, but still needs to access methods defined in `nexus-auth`. As a result, some `cfg(test)` methods are exposed as `pub` to make this work. This is one such example.